### DISCUSSION TITLE
How To Remove Padding in Title Bar When Firefox is Maximized?

### DISCUSSION THREAD

**Discussion by sapphired18** [black7375/Firefox-UI-Fix#844] [Q&A]
I am using vertical tab mode (with sidetabs) with this options
```
userChrome.hidden.tabbar -> true
userChrome.autohide.sidebar -> true
userChrome.sidebar.overlap -> true
userChrome.hidden.sidebar_header -> true
```

And this is what I mean
![image](https://github.com/black7375/Firefox-UI-Fix/assets/58063311/fb287c27-07f9-4b98-bf2a-7fe94a2abc6c)

The window controls also not aligned with other toolbar icons. and there's another orange color above the toolbar that I want to remove it

**Comment by black7375** [comment]
I think it's a bug, and I'll look into it further over the weekend.
